src/wmrc/main.py

```diff
@@ -151,113 +147,122 @@ def process():
         gis = arcgis.gis.GIS(config.AGOL_ORG, secrets.AGOL_USER, secrets.AGOL_PASSWORD)
 
         #: Do the work
-        module_logger.info('Loading data from Google Sheets...')
+        module_logger.info("Loading data from Google Sheets...")
         combined_df = _parse_from_google_sheets(secrets)
-        module_logger.info('Adding county names from SGID county boundaries...')
+        module_logger.info("Adding county names from SGID county boundaries...")
         with_counties_df = _get_county_names(combined_df, gis)
 
-        module_logger.info('Preparing data for truncate and load...')
+        module_logger.info("Preparing data for truncate and load...")
         proj_df = with_counties_df.copy()
         proj_df.spatial.project(4326)
-        proj_df.spatial.set_geometry('SHAPE')
-        proj_df.spatial.sr = {'wkid': 4326}
-        proj_df['last_updated'] = date.today()
-        proj_df = transform.DataCleaning.switch_to_datetime(proj_df, ['last_updated'])
+        proj_df.spatial.set_geometry("SHAPE")
+        proj_df.spatial.sr = {"wkid": 4326}
+        proj_df["last_updated"] = date.today()
+        proj_df = transform.DataCleaning.switch_to_datetime(proj_df, ["last_updated"])
         proj_df = transform.DataCleaning.switch_to_float(
-            proj_df, [
-                'latitude',
-                'longitude',
-                'tons_of_material_diverted_from_',
-                'gallons_of_used_oil_collected_for_recycling_last_year',
-            ]
+            proj_df,
+            [
+                "latitude",
+                "longitude",
+                # "tons_of_material_diverted_from_",
```

Comment:
Did you intend to leave this line commented out?

Reply:
Yes, this is part of the transition to salesforce and will probably go away in the next PR. For now, it's a reminder that removing this field was a temporary fix.